pdata/internal/metrics.go

```diff
@@ -183,6 +183,54 @@ func (at MetricAggregationTemporality) String() string {
 	return otlpmetrics.AggregationTemporality(at).String()
 }
 
+// Flags returns the flags associated with this NumberDataPoint.
+// Deprecated: [v0.57.0] Use FlagsStruct instead.
+func (ms NumberDataPoint) Flags() MetricDataPointFlags {
```

Comment:
We should deprecate MetricDataPointFlags as well.

Reply:
Does the alias cover it, or do you want me to deprecate the internal?

https://github.com/open-telemetry/opentelemetry-collector/blob/eb47b38b9ecec338668204b9d8d2f72851a670f3/pdata/pmetric/alias.go#L63

Reply:
Missed that, it works